addons/docs/src/frameworks/common/config.ts

```diff
@@ -1,12 +1,8 @@
-import { enhanceArgTypes } from './enhanceArgTypes';
-
 export const parameters = {
   docs: {
     inlineStories: false,
     getContainer: async () => (await import('../../blocks')).DocsContainer,
     getPage: async () => (await import('../../blocks')).DocsPage,
-    iframeHeight: 100,
+    iframeHeight: 110,
```

Comment:
Any particular reason for this change?

Reply:
Nope, good catch. I was trying to debug something and wanted a more unique number. Will change it back!